packages/generators/utils/add/index.ts

```diff
@@ -0,0 +1,64 @@
+import PROP_TYPES from "@webpack-cli/utils/prop-types";
```

Comment:
Other improvements is nice, cause CLI in add throws if dev-server is not a dependency, so you can keep the rest of the changes

Reply:
I didn't get what you want to say here?

Reply:
Things that has been changed in logic not related to the transformation w/AST is fine

Reply:
Things that has been changed in logic not related to the transformation w/AST is fine

Reply:
It's all good is what I'm saying. We need to look at the merge logic first in the ast part and then we can look at this, but it looks ok